x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/list_common/exceptions_utility/exceptions_utility.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import type { FC } from 'react';
+import styled from 'styled-components';
+import { FormattedMessage } from '@kbn/i18n-react';
+import type { Pagination } from '@elastic/eui';
+import { EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
+
+import {
+  UtilityBar,
+  UtilityBarSection,
+  UtilityBarGroup,
+  UtilityBarText,
+} from '../../../../../common/components/utility_bar';
+import { FormattedRelativePreferenceDate } from '../../../../../common/components/formatted_date';
+
+const StyledText = styled.span`
+  font-weight: bold;
+`;
+
+const MyUtilities = styled(EuiFlexGroup)`
+  height: 50px;
+`;
+
+const StyledCondition = styled.span`
+  display: inline-block !important;
+  vertical-align: middle !important;
+`;
+interface ExceptionsUtilityComponentProps {
+  showingTextDataTestSubj?: string;
+  lastUpdatedTextDataTestSubj?: string;
```

Comment:
What about just having user pass in one `dataTestSubj` and then for each of those components doing something like `data-test-subj={`${dataTestSubj}-showingText`}` and `data-test-subj={`${dataTestSubj}-lastUpdatedText`}`?

Reply:
Good point!

Reply:
As per our Tech meeting, we decided that this component will be added under the `exception-list-detail` page and passed as a prop to the `Exception-Items` component as it needs all of its components to be moved as well to the package